bugsnag-android-core/src/androidTest/resources/stackframe_numbers.json

```diff
@@ -0,0 +1,442 @@
+      "type": "c"
+    }
+  ],
+  "breadcrumbs": [
+    {
+      "name": "MainActivity",
+      "timestamp": "2018-10-19T18:56:13Z",
+      "type": "navigation",
+      "metaData": {
+        "ActivityLifecycle": "onStart()"
+      }
+    },
+    {
+      "name": "MainActivity",
+      "timestamp": "2018-10-19T18:56:13Z",
+      "type": "navigation",
+      "metaData": {
+        "ActivityLifecycle": "onResume()"
+      }
+    },
+    {
+      "name": "RINGER_MODE_CHANGED",
+      "timestamp": "2018-10-19T18:56:13Z",
+      "type": "state",
+      "metaData": {
+        "Extra": "RINGER_MODE_CHANGED: 2",
+        "Intent Action": "android.media.RINGER_MODE_CHANGED"
+      }
+    }
+  ],
+  "context": "MainActivity",
+  "severity": "warning",
+  "unhandled": true,
+  "severityReason": {
+    "type": "signal",
+    "attributes": {
+      "signalType": "SIGSEGV"
+    }
+  },
+  "app": {
+    "version": "1.1.14",
+    "id": "com.bugsnag.android.mazerunner",
+    "type": "android",
+    "releaseStage": "beta4",
+    "versionCode": 34,
+    "buildUUID": "2338403b-4ca1-40f5-bdee-3b82c2cb305a",
+    "duration": 169,
+    "durationInForeground": 0,
+    "inForeground": true
+  },
+  "metaData": {
+    "app": {
+      "packageName": "com.bugsnag.android.mazerunner",
+      "versionName": "1.1.14",
+      "activeScreen": "MainActivity",
+      "name": "MazeRunner",
+      "lowMemory": false
+    },
+    "device": {
+      "osBuild": "sdk_phone_armv7-eng 5.0.2 LSY66K 3079185 test-keys",
+      "apiLevel": 21,
+      "brand": "O-Matic",
+      "emulator": false,
+      "jailbroken": true,
+      "locale": "en_US",
+      "locationStatus": "allowed",
+      "networkAccess": "cellular",
+      "dpi": 160,
+      "screenDensity": 1,
+      "screenResolution": "480x320",
+      "time": "2018-10-19T18:56:13Z"
+    },
+    "customer": {
+      "name": "Acme Co",
+      "id": 23423,
+      "accountee": false
+    }
+  },
+  "device": {
+    "osName": "android",
+    "id": "9659cc1f-fec8-47a3-8ad2-0e3c7e84c24d",
+    "osVersion": "5.0.2",
+    "manufacturer": "Hudan",
+    "model": "sdk_phone_armv7",
+    "orientation": "portrait",
+    "totalMemory": 134217728,
+    "cpuAbi": [
+      "armeabi-v7a",
+      "armeabi"
+    ]
+  },
+  "session": {
+    "id": "225bcada-e0c8-15a0-0bba-0e3c7f43c13f",
+    "startedAt": "2018-10-19T18:56:13Z",
+    "events": {
+        "handled": 2,
+        "unhandled": 1
+    },
+    "user": {
+      "id": "9659cc1f-fec8-47a3-8ad2-0e3c7e84c24d",
+      "name": "Rasputin",
+      "email": "ras@example.com"
+    }
+  },
+  "user": {
+    "id": "9659cc1f-fec8-47a3-8ad2-0e3c7e84c24d",
+    "name": "Rasputin",
+    "email": "ras@example.com"
+  },
+  "groupingHash": "400-b",
+  "threads": [
+    {
+      "id": 1,
+      "name": "main",
+      "type": "android",
+      "stacktrace": [
+        {
+          "method": "dalvik.system.VMStack.getThreadStackTrace",
+          "file": "VMStack.java",
+          "lineNumber": -2
```

Comment:
Even though this is deserialised using the same method, it would be worth putting some assertions in for thread stacktrace line numbers.